pages/1.12/storage/persistent-volume/index.md

```diff
@@ -38,7 +38,7 @@ Configure a persistent volume with the following options:
 ```
 
 - `containerPath`: The path where your application will read and write data. This must be a single-level path relative to the container; it cannot contain a forward slash (`/`). (`"data"`, but not `"/data"`, `"/var/data"` or `"var/data"`).
-- `mode`: The access mode of the volume. Currently, `"RW"` is the only possible value and will let your application read from and write to the volume.
+- `mode`: The access mode of the volume. Currently, `"RW"` and `"RO"` are the only possible values and will let your application read from and write to the volume.
```

Comment:
"RW" will let your application read from and write to the volume, and "RO" will only let your application read from the volume.